composer.json

```diff
@@ -4,14 +4,15 @@
   "homepage": "https://github.com/automattic/vip-go-skeleton",
   "license": "GPL-2.0-or-later",
   "require": {
-    "php": ">= 7.2"
+    "php": ">= 7.4"
   },
   "require-dev": {
-    "automattic/vipwpcs": ">=0.4 <1.0",
-    "dealerdirect/phpcodesniffer-composer-installer": "^0.4.4",
+    "automattic/vipwpcs": "^2.3",
+    "dealerdirect/phpcodesniffer-composer-installer": "^0.7",
     "phpcompatibility/phpcompatibility-wp": "^2",
-    "squizlabs/php_codesniffer": "^3.3.2",
-    "wp-coding-standards/wpcs": "^1.2"
+    "squizlabs/php_codesniffer": "^3.5.5",
+    "wp-coding-standards/wpcs": "^2.3",
+    "sirbrillig/phpcs-variable-analysis": "^2.11.1"
```

Comment:
Since VIPCS pulls these in, can we avoid putting these direct dependency versions here?